WordPress/Classes/Utility/CoreDataManager.swift

```diff
@@ -1,10 +1,19 @@
 /// Handles the core data stack for the whole app
 class CoreDataManager: CoreDataStack {
 
-    static let shared = CoreDataManager()
+    // MARK: - Singleton
+
+    /// The shared instance.
+    ///
+    static var shared: CoreDataManager = CoreDataManager(fileLocationManager: CoreDataFileLocationManager.production())
```

Comment:
any reason why this changed from `let` to `var`?

Reply:
Just something in need of a cleanup.  Good catch.